build/bazel/remote/execution/v0.2/remote_execution.proto

```diff
@@ -256,6 +256,10 @@ service ContentAddressableStorage {
   // [ContentAddressableStorage][build.bazel.remote.execution.v0_2.ContentAddressableStorage]
   // (CAS). The server will enumerate the `Directory` tree recursively and
   // return every node descended from the root.
+  // The GetTreeRequest.page_token parameter can be used to skip ahead in
```

Comment:
Nit: either include a blank line to separate paragraphs, or wrap the lines together.